### PR TITLE
Bump up operator version

### PR DESCRIPTION
**Pull request by rensyct**
# Description
This PR
1. Upgrades the operator version
2. Updates the CSVs with the changes that had to be done for postings
3. Updates the annotations with the changes that had to be done for postings

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/254 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

### Validation Screenshots
image

### CODE REVIEW

**Review by panigs7 — DISMISSED** [review]

**Review by rajendraindukuri — DISMISSED** [review]

File: config/samples/storage_v1_csm_powerscale.yaml
Comment (rensyct) [reply config/samples/storage_v1_csm_powerscale.yaml:7]: I would prefer to merge this now if all are fine with that.. Once the configMap changes are in, again all manifests have to be updated wrt crd version and remoteRepo... I will cover those again as part of one another PR that is planned to be delivered next sprint

Comment (rensyct) [reply config/samples/storage_v1_csm_powerscale.yaml:7]: I can wait till the configMap and version related changes are merged to main

**Review by panigs7 — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]

**Review by tdawe — APPROVED** [review]

**Pull request merged**
